src/program/managers/event_manager.py

```
@@ -1,5 +1,7 @@
 import os
+import sys
```

Ruff (0.8.0)

`sys` imported but unused

Remove unused import: `sys`

(F401)